src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseData.java

```diff
@@ -468,6 +468,7 @@ public class CaseData extends CaseDataParent {
     private final String dateOfDeathType;
     private final String resolveStopState;
     private final String transferToState;
+    private final String resolveCaveatStopState;
```

Comment:
should we handle this as an enum similar to applicationType: [ApplicationType](https://github.com/hmcts/probate-back-office/blob/master/src/main/java/uk/gov/hmcts/probate/model/ApplicationType.java) - i realize that everything here is a string, but that makes it incredibly annoying to work with these objects